src/features/surveys/components/ResponseStatsCards.tsx

```diff
@@ -531,7 +531,22 @@ const OptionsStatsCard = ({
   );
 };
 
-const WordCloud = memo(ReactWordcloud);
+interface WordData {
+  text: string;
+  value: number;
+}
+const WordCloudFixedValueGenerator = () => 0.5;
+
+export function makeDeterministicRNG(seed: number) {
+  return function random() {
+    seed = (seed * 1664525 + 1013904223) >>> 0;
+    return seed / 0xffffffff;
```

Comment:
I personally cannot understand what code does and how these numbers were chosen, so it would be difficult for me to take on a task that would involve interacting with this bit of code.  I assume these numbers are selected for good reason and I see your skill and knowledge here :D 

To make the code a bit less obsucre, if you want, it would be nice to save the numbers in named constants? and mabye add a comment above line 543 to clarify what this line of code does? this is the first time i see the `>>>` operator used in our code base and although I kind of understand now that I've read up on what it is, it would be of great help to have a short sentence explaining why this is chosen here.